galaxy.yml

```diff
@@ -1,10 +1,16 @@
 ---
-authors:
-  - Ansible Network Community (ansible-network)
-dependencies:
-  "ansible.netcommon": "*"
-license_file: LICENSE
+namespace: community
```

Comment:
I've renamed the repo to `https://github.com/ansible-collections/community.skydive`, thanks

Reply:
Thanks @gundalow !